e2e/package.json

```diff
@@ -38,31 +38,31 @@
   },
   "dependencies": {},
   "devDependencies": {
-    "@types/expect-puppeteer": "^4.4.1",
```

Comment:
Can you comment on your methodology for making these upgrades? If it's a regular maintenance activity, it may be worth lightly documenting the process somewhere in the readme.